packages/core/addon/components/cell-renderers/dimension.js

```diff
@@ -23,29 +23,38 @@ export default Component.extend({
    */
   classNames: ['table-cell-content', 'dimension'],
 
+  /**
+   * @property {String} name - dimension name
+   */
+  name: computed('column.attributes.name', function() {
+    return get(this, 'column.attributes.name');
+  }),
+
   /**
    * @property {String} descField - field to find the description
    */
-  descField: computed('column.field.dimension', function() {
-    return get(this, 'column.field.dimension') + '|desc';
+  descField: computed('name', function() {
+    return get(this, 'name') + '|desc';
   }),
 
   /**
    * @property {String} idField - field to find id
    */
-  idField: computed('column.field.dimension', function() {
-    return get(this, 'column.field.dimension') + '|id';
+  idField: computed('name', function() {
+    return get(this, 'name') + '|id';
   }),
 
   /**
    * @property {Object} dimensionField - field to find dimension field
    */
-  dimensionField: computed('column.field.dimension', 'column.field.field', function() {
-    if (!get(this, 'column.field.field')) {
+  dimensionField: computed('name', 'column.attributes.field', function() {
+    let field = get(this, 'column.attributes.field');
+
+    if (!field) {
```

Comment:
Could we simplify this to ``return field ? `${get(this, 'name')}|${field}` : null;`` ?